Rubrik/Public/Get-RubrikVM.ps1

```diff
@@ -27,6 +27,10 @@ function Get-RubrikVM
       .EXAMPLE
       Get-RubrikVM -Relic
       This will return all removed virtual machines that were formerly protected by Rubrik.
+
+      .EXAMPLE
+      Get-RubrikVM -id myserver01 -DetailedObject
```

Comment:
I believe this example should use the `Name` param.

```
Get-RubrikVM -Name 'Server1' -DetailedObject
```

Reply:
Cool I updated the description and added a progress bar for DetailedObject queries